Source/WebKit/WebProcess/Plugins/PDF/WKAccessibilityPDFDocumentObject.mm

```diff
@@ -231,12 +263,10 @@ - (void)gotoDestination:(PDFDestination *)destination
     if (!plugin)
         return;
 
-    NSInteger pageIndex = [_pdfDocument indexForPage:[destination page]];
-    NSRect rect = [[self accessibilityChildren][0] accessibilityFrame];
-    WebCore::IntPoint point = { 0, static_cast<int>(rect.size.height * pageIndex) + 1 };
+    int pageIndex = (int) [_pdfDocument indexForPage:[destination page]];
```

Comment:
You can rewrite this with `PDFDocumentLayout::PageIndex`
```suggestion
    PDFDocumentLayout::PageIndex pageIndex = [_pdfDocument indexForPage:[destination page]];
```

Reply:
This will fit in nicely when you also let `UnifiedPDFPlugin::accessibilityScrollToPage` accept a `PDFDocumentLayout::PageIndex` argument.

Reply:
It seems like this suggestion didn't stick? Mind addressing this again?

```cpp
PDFDocumentLayout::PageIndex pageIndex = [_pdfDocument indexForPage:[destination page]];
callOnMainRunLoop([plugin, pageIndex] {
    plugin->accessibilityScrollToPage(pageIndex);
});
```